src/csharp/Grpc.nuspec

```diff
@@ -5,7 +5,7 @@
     <title>gRPC C#</title>
     <summary>C# implementation of gRPC - an RPC library and framework</summary>
     <description>C# implementation of gRPC - an RPC library and framework. See project site for more info.</description>
-    <version>0.5.0</version>
+    <version>0.5.0.1</version>
```

Comment:
Then it would make sense to make all the other packages to 0.5.1 as well so that their version stays in sync. Like this we have packages for release 0.5.0 and one of them has been republished because there was a problem (but a packaging problem, the code stays 100% the same).